.ruby-version

```diff
@@ -0,0 +1 @@
+2.1.7
```

Comment:
Can we document where this specification comes from? Add a comment with a link to the `github-pages` ruby version?